quotes.json

```diff
@@ -2523,6 +2529,18 @@
     "author": "Friedrich von Schiller",
     "text": "If you want to study yourself, look into the hearts of other people. If you want to study other people, look into your own heart."
   },
+  {
+    "author": "Fyodor Dostoyevsky",
+    "source": "https://www.goodreads.com/author/quotes/3137322.Fyodor_Dostoyevsky",
+    "tags": "pain, suffering, life",
+    "text": "Pain and suffering are always inevitable for a large intelligence and a deep heart. The really great men must, I think, have great sadness on earth."
+  },
+  {
+    "author": "Fyodor Dostoyevsky",
+    "source": "https://www.goodreads.com/author/quotes/3137322.Fyodor_Dostoyevsky",
+    "tags": "soul, children, healing",
+    "text": "The soul is healed by being with children."
```

Comment:
Totally agree with this and it's a good reminder to spend more time with kids. Inspiring and positive. 👍